src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,90 @@
+import { FC } from 'react';
+import cn from 'classnames';
+import { useParams } from 'react-router-dom';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink';
+// import { SearchLink } from '../SearchLink/SearchLink';
+import { TableHeader } from '../TableHeader';
+
+interface Props {
+  people: Person[];
+}
+
+export const PeopleTable: FC<Props> = ({ people }) => {
+  const { personSlug } = useParams();
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>
+            <TableHeader title="Name" />
+          </th>
+
+          <th>
+            <TableHeader title="Sex" />
+          </th>
+
+          <th>
+            <TableHeader title="Born" />
+          </th>
+
+          <th>
+            <TableHeader title="Died" />
+          </th>
+
+          <th>Mother</th>
+          <th>Father</th>
```

Comment:
can't you make use of your TableHeader  component?

Reply:
I can't because TableHeader component add buttons, so I use it only for elements that can be filtered. But these two headers don't have buttons according to example
![image](https://github.com/mate-academy/react_people-table-advanced/assets/87998640/45931978-246f-480f-ae92-52f8a8f7c795)